### PR TITLE
eric test

### PR DESCRIPTION
**Pull request by carsongee**
You really want this, it is amazing.

### CODE REVIEW

**Comment by carsongee** [comment]
I am not sure about this whole thing

File: presentation.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 title: Git & Github Quickstart
 output: index.html
-
+#Eric is playing with his branch
```
Comment (carsongee) [inline presentation.md]: This line is terrible

Comment (erb3742) [inline presentation.md]: But it is the best code I ever wrote.

File: presentation.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 title: Git & Github Quickstart
 output: index.html
-#Eric is playing with his branch
+#Eric is playing with his branch..
```
Comment (carsongee) [inline presentation.md:3]: Periods wow, this is great

**Comment by carsongee** [comment]
🐤

**Comment by carsongee** [comment]
🚳

**Pull request merged**